app/components/procedure/instructeurs_options_component/instructeurs_options_component.html.haml

```diff
@@ -54,15 +54,27 @@
     data: { controller: 'enable-submit-if-checked' },
     url: wizard_admin_procedure_groupe_instructeurs_path(@procedure) do |f|
 
-    %div{ data: { 'action': "click->enable-submit-if-checked#click" } }
+    %h1 Configuration du routage
+    %h2 Choix du type de routage
+
+    .card.fr-pb-0{ data: { 'action': "click->enable-submit-if-checked#click" } }
+      %p.fr-mb-0 Routage
       = render Dsfr::RadioButtonListComponent.new(form: f,
         target: :state,
         buttons: [ { label: 'À partir d’un champ', value: 'routage_simple', hint: 'crée les groupes en fonction d’un champ du formulaire' } ,
-          { label: 'Avancé', value: 'routage_custom', hint: 'libre à vous de créer et de configurer les groupes' }]) do
-        %h1 Choix du type de routage
+          { label: 'Avancé', value: 'custom_routing', hint: 'libre à vous de créer et de configurer les groupes' }])
 
-      %ul.fr-btns-group.fr-btns-group--inline-sm
-        %li
-          = link_to 'Retour', options_admin_procedure_groupe_instructeurs_path(@procedure), class: 'fr-btn fr-btn--secondary'
-        %li
-          %button.fr-btn{ disabled: true,  data: { 'enable-submit-if-checked-target': 'submit' } } Continuer
+
+    %ul.fr-btns-group.fr-btns-group--inline-sm
+      %li
+        = link_to 'Annuler', options_admin_procedure_groupe_instructeurs_path(@procedure), class: 'fr-btn fr-btn--secondary'
+      %li
+        %button.fr-btn{ disabled: true,  data: { 'enable-submit-if-checked-target': 'submit' } } Continuer
+- if params[:state] != 'choix'
+  .padded-fixed-footer
+    .fixed-footer
+      .fr-container
+        %ul.fr-btns-group.fr-btns-group--inline-md.fr-ml-0
+          %li
+            = link_to admin_procedure_path(id: @procedure), class: 'fr-link fr-icon-arrow-left-line fr-link--icon-left fr-mb-2w fr-mr-2w' do
```

Comment:
Alors, j'ai découvert [ça](https://guides.rubyonrails.org/routing.html#creating-paths-and-urls-from-objects) et je pense que ça a changé ma vie

```suggestion
            = link_to [:admin, @procedure], class: 'fr-link fr-icon-arrow-left-line fr-link--icon-left fr-mb-2w fr-mr-2w' do
```

a consommer sans modération dans toutes tes vues.

Reply:
trop bien 👍